content/en/users/compute/cloud-compute/auth/_index.md

```diff
@@ -151,5 +151,55 @@ export OS_TOKEN=$(fedcloud openstack --site <NAME_OF_SITE> --vo <NAME_OF_VO> \
 
 ### Using ssh-oidc
 
-[ssh-oidc](https://github.com/EOSC-synergy/ssh-oidc) is a set of tools that
-allows ssh with OIDC.
+[ssh-oidc](https://github.com/EOSC-synergy/ssh-oidc) is a collection of
+tools that enable ssh using OpenID-Connect access tokens.
+
+Most of the individual tools work standalone, but they may be combined to
+establish complex installations with dynamic account provisioning in
+multi-OP and multi-VO contexts.
```

Comment:
what is multi-OP?

Reply:
Supporting more than a single OP (OIDC-Provider). I.e. it's possible to support EGI-Checkin and Indigo-IAM at the same time.
Shall I extend that description?

Reply:
yes, please, not sure we are using this term anywhere in the doc, we are Check-in powered you know ;)